config/jobs/kubernetes-csi/external-snapshotter/external-snapshotter-config.yaml

```diff
@@ -21,6 +21,8 @@ presubmits:
         args:
         - ./.prow.sh
         env:
+        - name: CSI_PROW_KUBERNETES_VERSION
+          value: 1.15
```

Comment:
This has to be in quotation marks, otherwise it gets decoded as number, leading to the test failure (https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/test-infra/13434/pull-test-infra-bazel/1149800668769292288):
```
ould not load config: error unmarshaling ../../../config/jobs/kubernetes-csi/external-snapshotter/external-snapshotter-config.yaml: error unmarshaling JSON: while decoding JSON: json: cannot unmarshal number into Go struct field EnvVar.value of type string================================================================================
FAIL: //config/tests/jobs:go_default_test (see /bazel-scratch/.cache/bazel/_bazel_root/05618a594cb3499a4b912817865867cd/execroot/io_k8s_test_infra/bazel-out/k8-fastbuild/testlogs/config/tests/jobs/go_default_test/test.log) 
```

Also see the other comment about keeping this as `1.15.0`.

Reply:
done